src/chart.zoom.js

```diff
@@ -201,8 +201,13 @@ function panIndexScale(scale, delta, panOptions) {
 
 function panTimeScale(scale, delta, panOptions) {
 	var options = scale.options;
-	options.time.min = rangeMinLimiter(panOptions, scale.getValueForPixel(scale.getPixelForValue(scale.firstTick) - delta));
-	options.time.max = rangeMaxLimiter(panOptions, scale.getValueForPixel(scale.getPixelForValue(scale.lastTick) - delta));
+	
+	var newMax = rangeMaxLimiter(panOptions, scale.getValueForPixel(scale.getPixelForValue(scale.lastTick) - delta));
+	var newMin = rangeMinLimiter(panOptions, scale.getValueForPixel(scale.getPixelForValue(scale.firstTick) - delta));
+	if (newMax != options.time.max && newMin != options.time.min) {
```

Comment:
I just want to make sure I got everything right. So this:
```
newMax != options.time.max && newMin != options.time.min
```
will have to change to
```
newMax > options.time.max && newMin > options.time.min
|| newMax < options.time.max && newMin < options.time.min
```
options.time.max and options.time.min seem to be undefined at first. Added this check and ended up with:
```
if (!options.time.max || !options.time.min     //Check is not working on first iteration
     || newMax > options.time.max && newMin > options.time.min
     || newMax < options.time.max && newMin < options.time.min) {
	
        [...]

}
```

Would this be ok?